### DISCUSSION TITLE
可以做云游戏服务端架构吗

### DISCUSSION THREAD

**Discussion by denghai** [cloudwu/skynet#1040] [Q&A]
https://github.com/bajdcc/GameFramework，可以把gui、图形渲染、物理引擎、虚拟机等等考虑做成单独的模块吗，如果能用unity或者cocos的虚拟机就更方便了

**Comment by yomunsam** [comment]
咱能不能单纯的好奇下，您是计算机专业的吗，大一课程及格了没

**Comment by sishui** [comment]
做成一键生成游戏 更酸爽
